src/FTL.xml

```diff
@@ -0,0 +1,121 @@
+<SPDX name="Freetype Project License" identifier="FTL" osi-approved="false">
+  <urls>
+    <url>http://freetype.fis.uniroma2.it/FTL.TXT</url>
+  </urls>
+  <notes>This license was released 27 Jan 2006</notes>
+  <license>
+    <title>
+      <p>The FreeType Project LICENSE</p>
+      <p>2006-Jan-27</p>
+    </title>
+    <copyright>
+      <p>Copyright 1996-2002, 2006 by David Turner, Robert Wilhelm, and Werner Lemberg</p>
+    </copyright>
+    <optional>
+      <p>Introduction</p>
+      <p>The FreeType Project is distributed in several archive packages; some of them may contain, in addition to
+         the FreeType font engine, various tools and contributions which rely on, or relate to, the FreeType
+         Project.</p>
+      <p>This license applies to all files found in such packages, and which do not fall under their own explicit
+         license. The license affects thus the FreeType font engine, the test programs, documentation and
+         makefiles, at the very least.</p>
+      <p>This license was inspired by the BSD, Artistic, and IJG (Independent JPEG Group) licenses, which all
+         encourage inclusion and use of free software in commercial and freeware products alike. As a
+         consequence, its main points are that:</p>
+      <p>o We don't promise that this software works. However, we will be interested in any kind of bug reports.
```

Comment:
The 'o' characters are actually bullets and so these are the related text ought to be converted into list tags. Pull request submitted.